infra/testing/server/routes/sw-bundle.js

```diff
@@ -0,0 +1,44 @@
+/*
+  Copyright 2019 Google LLC
+
+  Use of this source code is governed by an MIT-style
+  license that can be found in the LICENSE file or at
+  https://opensource.org/licenses/MIT.
+*/
+
+const {rollup} = require('rollup');
+const replace = require('rollup-plugin-replace');
+const resolve = require('rollup-plugin-node-resolve');
+const multiEntry = require('rollup-plugin-multi-entry');
+
+
+const match = '/test/:package/*/sw-bundle.js';
+let cache;
+
+async function handler(req, res) {
+  const bundle = await rollup({
+    input: `./test/${req.params.package}/sw/**/test-*.mjs`,
+    plugins: [
+      multiEntry(),
+      resolve(),
+      replace({
+        'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV),
+      }),
+    ],
+    cache,
+  });
+
+  // Update the cache so subsequent bundles are faster.
```

Comment:
We definitely don't need to kill the test server each time. I confirmed that it both does pick up the changes as well as *seems* to build faster the second time.

It's not clear to me how much it's saving -- you can look at the cache object and see what it's storing, and it's not storing things like last modified time, so it may still be reading the file again.